services/autoscaling/src/simcore_service_autoscaling/modules/auto_scaling_core.py

```diff
@@ -85,6 +85,28 @@ async def _analyze_current_cluster(
         docker_nodes, existing_ec2_instances
     )
 
+    # analyse pending ec2s, check if they are pending since too long
+    now = arrow.utcnow().datetime
+    broken_ec2s = [
+        instance
+        for instance in pending_ec2s
+        if (now - instance.launch_time)
+        > app_settings.AUTOSCALING_EC2_INSTANCES.EC2_INSTANCES_MAX_START_TIME
+    ]
+    if broken_ec2s:
+        _logger.error(
+            "Detected broken EC2 instances that never joined the cluster after %s: %s\n"
+            "TIP: if this happens very often the time to start an EC2 might have increased or "
+            "something might be wrong with the used AMI and/or boot script in which case this"
+            " would happen all the time. Please check",
+            app_settings.AUTOSCALING_EC2_INSTANCES.EC2_INSTANCES_MAX_START_TIME,
```

Comment:
maybe also print, after `Please check`, the value of the env var as a string "EC2_INSTANVE_MAX_START_TIME" so that it can be more easily searched for.

Reply:
it is printed. what do you mean?